calculator.c

```diff
@@ -0,0 +1,80 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
```

Comment:
Be considerate about the #includes. Let's not use includes if not absolutely necessary. In this case, stdbool.h is not absolutely necessary.